client/components/recycleAndReuseComponents/AddItem.js

```diff
@@ -18,6 +18,7 @@ const AddItem = ({ setNextStep, data, setItems }) => {
 				data={data}
 				setItems={setItems}
 				openDialog={() => willTriggerDialog(true)}
+				willTriggerDialog={willTriggerDialog}
```

Comment:
revert all other changes - only main logic change should remain (as mentioned below)